build.rs

```diff
@@ -1,17 +1,32 @@
+#[cfg(feature = "rt")]
+use std::env;
+
+#[cfg(feature = "rt")]
+const DEFAULT_HEAP_SIZE: usize = 16 * 1024;
+#[cfg(feature = "rt")]
+const DEFAULT_STACK_SIZE: usize = 16 * 1024;
+
 #[cfg(feature = "rt")]
 fn main() {
     use imxrt_rt::{Family, FlexRamBanks, Memory, RuntimeBuilder};
 
+    let heap_size = env::var("TEENSY4_HEAP_SIZE")
```

Comment:
Part of me wonders if we should require this to be in units of KiB, although the docs do allow any number of bytes.

Reply:
The code renormalizes this to multiples of 8, and someone could really want a 48 byte stack if they were being really precise.

Reply:
If we were really nice, we could parse 8K as 8192.  I don't see a reason to parse 1M though.